tasks/code_sync_status.rb

```diff
@@ -0,0 +1,93 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+
+# Puppet Task Name: code_sync_status
+require 'net/https'
+require 'uri'
+require 'json'
+require 'puppet'
+
+# CodeSyncStatus task class
+class CodeSyncStatus
+  def initialize(params)
+    @params = params
+  end
+
+  def execute!
+    puts syncstatus.to_json
+  end
+
+  def https
```

Comment:
While the word `https` is not a reserved word perse in Ruby, this could get confusing when your class is used by other classes. Or this could be interpreted as if you are overwriting the `https` method from probably `'net/https'`

Should we rename this method to `https_client`?

Reply:
good call